client/src/components/OnboardUser.js

```diff
@@ -0,0 +1,233 @@
+import React, { useState } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import NavBar from './NavBar';
+import { ReactComponent as Logo } from '../img/Logo_CORONATRACKER_Logo.svg';
+import { ReactComponent as TextLogo } from '../img/Logo_CORONATRACKER_Text_Logo.svg';
+import profileImg from '../img/profile.png';
+
+const useStyles = makeStyles(theme => ({
+  //the styles goes here as an object
+  root: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    justifyContent: 'space-evenly',
+    margin: '2em 0 2em 0',
+  },
+  button: {
+    color: 'white',
+    backgroundColor: 'red',
+    fontWeight: 'bold',
+    alignSelf: 'center',
+    height: '4vh',
+    borderRadius: '5px',
+    margin: '0 0 0 0',
+    maxWidth: '300px',
+    minWidth: '42%',
+  },
+  header: {
+    width: 325,
+    display: `inline-block`,
+    marginBottom: '.5em',
+    fontSize: '.5em',
+  },
+  profile: {
+    width: '100px',
+    height: '140px',
+    marginBottom: '.5em',
+    fontSize: '1em',
+  },
+  image: {
+    width: '100%',
+  },
+  inputArea: {
+    display: 'flex',
+    flexDirection: 'row',
+    marginBottom: '.5vh',
+  },
+  inputField: {
+    width: '48%',
+    textAlign: 'center',
+    maxWidth: '180px',
+    color: 'white',
+    backgroundColor: 'red',
+    height: '4vh',
+    justifyContent: 'space-between',
+    border: 'none',
+    borderRadius: '5px',
+    '&::placeholder': {
+      color: 'white',
+    },
+
+    '&:-ms-input-placeholder': {
+      color: 'white',
+    },
+
+    '&::-ms-input-placeholder': {
+      color: 'white',
+    },
+  },
+  inputFieldLabel: {
+    textAlign: 'right',
+    width: '48%',
+  },
+  location: {
+    border: 'none',
+    marginBottom: '10px',
+    textAlign: 'center',
+    color: 'white',
+    backgroundColor: 'red',
+    width: '33%',
+    maxWidth: '60px',
+    fontSize: '1em',
+    borderRadius: '5px',
+    fontFamily:
+      '-apple-system, BlinkMacSystemFont, avenir next, avenir, helvetica neue, helvetica, ubuntu, roboto, noto, segoe ui, arial,',
+    '&::placeholder': {
+      color: 'white',
+    },
+
+    '&:-ms-input-placeholder': {
+      color: 'white',
+    },
+
+    '&::-ms-input-placeholder': {
+      color: 'white',
+    },
+  },
+}));
+
+const blankForm = {
+  firstName: '',
+  age: '',
+  sex: '',
+  city: '',
+  state: '',
+  zip: '',
+};
+
+export default ({ postNewUser }) => {
+  const [formState, setFormState] = useState(blankForm);
+  const handleChange = e => {
+    e.preventDefault();
+    setFormState({
+      ...formState,
+      [e.target.name]: e.target.value,
+    });
+  };
+  const classes = useStyles();
+  return (
+    <div className={classes.root}>
+      <div className={classes.header}>
+        <h4>
+          <b>Welcome to CoronaTracker!</b>
+        </h4>
```

Comment:
I think we should be defining HTML elements using Material-UI styling, so a `Typography variant='h4'` in this case

Reply:
I've got to be honest, this component was basically finished last night and I was pretty frustrated when @AdhamAH noted

"@fallon7284 thanks for working on it! Please try to keep in mind using Material-UI styling in doing this but use the jsx normal input tag."

because I had already done it in Material-UI components and those guidelines had not been noted in the issue.  I went back today and changed it all to jsx.  This feels like a nitpick, no?

Reply:
Okay, let's leave this for now. This is nitpicky and definitely doubles down on the need for an open, easily-accessible style guide. I'll reach out to you to see what action we can take regarding this.